public/views/signin.html

```diff
@@ -1,76 +1,84 @@
 <div ng-controller="IndexController">
+    <script>
+      // var closeOverlay = function (){
+      //   var sidenavOverlay = document.querySelector('.sidenav-overlay');
+      //   sidenavOverlay.style.opacity = '0';
+      //   sidenavOverlay.style.display = 'none';
+      // };
+      $('.sidenav').sidenav({
+        closeOnClick  : true
+      });
+    </script>
```

Comment:
@topseySuave I don't think you should comment out codes when you are pushing.
If you are certain that the code snippet you previously have used will not be useful anymore,
remove it from code you are pushing to production so other people working on the project will have a better experience following what is going on in the file.
But GREAT WORK!